src/components/binding/binding-panel.js

```diff
@@ -160,10 +160,10 @@ genotet.BindingPanel.prototype.initChrs_ = function() {
 
 /**
  * Adds a select2 for a new track.
+ * @param {number} trackIndex Index of the added track.
  * @private
  */
-genotet.BindingPanel.prototype.addTrack_ = function() {
-  var trackIndex = this.data.tracks.length - 1;
+genotet.BindingPanel.prototype.addTrack_ = function(trackIndex) {
```

Comment:
Starting from line 216 in binding-panel.js we have:
```javascript
this.data.tracks.forEach(function(track, index) {
    var ui = this.container.find('#genes #track-' + index);
    if (!ui.length) {
      this.addTrack_(index);
    }
    this.selectGenes_[index].val(track.fileName).trigger('change');
  }, this);
```
This is the only place we use addTrack_, and the track select list is constructed according to their order in this.data.tracks. Why did you think they could be in the middle of the list?

Haven't yet. I am going to have a try asap.